2232/3/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < Ohm::Model
+  attribute :text
+  attribute :author
+  attribute :rating
+end
```

Comment:
don't u want to add  a reference to the article

Reply:
The link to the article is stored in the Article module